lib/index.js

```diff
@@ -714,14 +717,20 @@ class Mcdev {
                 Util.logger.info(`\n               :: Executing the entity on all BUs for ${cred}`);
 
                 for (const bu in properties.credentials[cred].businessUnits) {
-                    await this._executeBU(cred, bu, selectedTypesArr, keys);
-                    counter_credBu++;
+                    result = await this._executeBU(cred, bu, selectedTypesArr, keys);
```

Comment:
this overwrites previous failures. please collect failures AND success in separate variables, then check if you found any errors